### PR TITLE
Clarify peristent disk behavior when a VM is deleted

### PR DESCRIPTION
**Pull request by ywei2017**
Provide clarity of persistent disk behavior when a VM is deleted. See [bug 660](https://github.com/cloudfoundry/docs-bosh/issues/660)

### CODE REVIEW

**Comment by h4xnoodle** [comment]
Thanks for your contribution.

I think we could further clarify that `bosh delete-vm` will not perform the same operation as an IaaS-level 'delete'. From Bosh's perspective, we don't really have control over what an IaaS decides to do for delete. This is out of the scope of Bosh's behaviours. In general, _all_ VM operations _should_ be performed by Bosh, and Bosh can recover well from a few situations that use non-Bosh operations to manipulate VMs.

Could you add a note on how `bosh delete-vm` will perform the expected way as per the original doc, but an IaaS-level delete will have IaaS-specific behaviours?

Thanks,
Rebecca, BOSH

**Comment by ywei2017** [comment]
@h4xnoodle Sure, no problem. See the updated P/R.

Thanks
Yansheng

File: content/persistent-disks.md [hunk 1]
```diff
@@ -3,7 +3,13 @@
 
 Instance groups may need to store persistent data.
 
-If you attach a persistent disk to a virtual machine and then stop, terminate, or delete the VM, your persistent disk data remains intact. Attaching the persistent disk to another VM allows you to access your data.
+If you attach a persistent disk to a virtual machine and then stop the VM (such as `bosh delete-vm`), your
```
Comment (h4xnoodle) [inline content/persistent-disks.md]: "stop" and "bosh delete-vm" aren't really the same thing here. `bosh stop` would likely be more what operators perceive as "stopped". 

Perhaps something like
> If you attach a persistent disk to a virtual machine and then delete the VM via `bosh delete-vm`, your persistent disk data remains intact.

**Comment by h4xnoodle** [comment]
@ywei2017 would you mind making the distinction between 'stop' and 'delete' as per the note?

**Comment by ywei2017** [comment]
@h4xnoodle Good idea. I just updated as suggested.

Thanks
Yansheng

**Comment by h4xnoodle** [comment]
Looks good! @mfine30 will end up merging.

**Pull request merged**